src/world/mod.rs

```diff
@@ -16,11 +16,49 @@ mod lazy;
 #[cfg(test)]
 mod tests;
 
+#[cfg(not(feature = "nightly"))]
```

Comment:
Could we add something like `"Note: enable nightly feature to get the component type printed out"`?